### DISCUSSION TITLE
Support for the Arducam Darksee IMX678

### DISCUSSION THREAD

**Discussion by ehcatl** [aaronwmorris/indi-allsky#1672] [General]
Hi @aaronwmorris:

Could you add support for the libcamera from Arducam named Darksee with the IMX678 sensor? 

I am trying with the IMX462 sensor but I am not quite sure if that one is going to work. 

Thanks,

**Comment by aaronwmorris** [comment]
Merged #1673 .  There are a few more details that need to be verified, such as the min/max gain, but that should get things started.
